shell/platform/embedder/embedder.h

```diff
@@ -539,6 +539,10 @@ typedef struct {
   // Flutter application (such as compiled shader programs used by Skia).
   // This is optional.  The string must be NULL terminated.
   const char* persistent_cache_path;
+
+  // If true, we'll only read the existing cache, but not write new ones.
+  bool is_persistent_cache_read_only;
```

Comment:
Instead of "we", please say "the engine". Also, we should only read this field if the `persistent_cache_path` is non-null.

Reply:
Changed "we" to "engine".

Shall we also allow read-only cache for the default cache path where persistent_cache_path is nullptr?

Reply:
I dont think so. On Linux, the default cache path is empty and the persistent cache wont be setup in such cases.

Reply:
Ok. Done.